python/PyQt/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ SET(PYQT_COMPAT_FILES
   QtSvg.py
   Qsci.py
   Qt.py
+  sip.py
```

Comment:
did you forget to add this file in your commit?

Reply:
why yes, i did...